bootstrap

```diff
@@ -406,7 +410,8 @@ swig_git_commit_hash=613ff08150e6264a5a4a964f2badeb6bba221398
 omega_common_commit_hash=f5921b74bc4bf899d9a043ee3ecdbcbfd4bacb7b
 letor_common_commit_hash=f5921b74bc4bf899d9a043ee3ecdbcbfd4bacb7b
 
-if [ ! -r .git ] ; then
+gitdir=`git rev-parse --git-dir`
+if [ ! -r $gitdir ] ; then
```

Comment:
`$gitdir` should really be `"$gitdir"` everywhere in case it contains spaces.

Reply:
yes, good point!  I'll definitely fix that up.